polkadot/node/network/dispute-distribution/src/sender/send_task.rs

```diff
@@ -277,13 +338,40 @@ async fn send_requests<Context, M: 'static + Send + Sync>(
 	ctx: &mut Context,
 	tx: NestingSender<M, TaskFinish>,
 	receivers: Vec<AuthorityDiscoveryId>,
+	rejected_deliveries: &HashMap<AuthorityDiscoveryId, (Timestamp, usize)>,
 	req: DisputeRequest,
 	metrics: &Metrics,
 ) -> Result<HashMap<AuthorityDiscoveryId, DeliveryStatus>> {
 	let mut statuses = HashMap::with_capacity(receivers.len());
 	let mut reqs = Vec::with_capacity(receivers.len());
+	let now = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs();
 
+	gum::trace!(
+		target: LOG_TARGET,
+		?rejected_deliveries,
+		"Sending dispute request to authorities."
+	);
 	for receiver in receivers {
+		if let Some((last_retry_time, num_retries)) = rejected_deliveries.get(&receiver) {
+			// Context for the backoff logic: https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/retry-backoff.html
+			if now < *last_retry_time + (2 ^ *num_retries as u64) {
```

Comment:
This per peer & per task back-off seems much less problematic. In any case, I would suggest to request an audit for these changes.